### ISSUE TITLE
 README.md seems not updated in the main version.

### ISSUE THREAD

**Issue by arminkollascheck** [OpenTrafficCam/OTVision#257]
Windows: 
"Click on .\OTVision\Install.bat (a venv will be created and packages from requirements.txt will be installed including PyTorch versions published on [pytorch.org](https://pytorch.org/get-started/locally/))"

But there is no Install.bat in the current version. Guess its the install.cmd?

Requirements:
"Python 3.9.x (add to PATH while installation)"

Is still the case? Seems like it uses Python 3.10.X. ("Python Version 3.10 is not installed in environment.")

**Comment by arminkollascheck** [comment]
Used the current main version.

**Comment by frunika** [comment]
True... We need to update all Readme files on all otc repos... argh. fun... not :D
